src/affexpr.jl

```diff
@@ -28,12 +28,20 @@ GenericAffExpr(a::GenericAffExpr) = GenericAffExpr(a.vars, a.coeffs, a.constant)
 
 coeftype(::GenericAffExpr{C,V}) where {C,V} = C
 
+# variables must be ∈ ℝ but coeffs can be ∈ ℂ
+Compat.adjoint(a::GenericAffExpr{<:Real}) = a
+function Compat.adjoint(a::GenericAffExpr)
+    GenericAffExpr(a.vars, [z' for z ∈ a.coeffs], a.constant')
```

Comment:
Could you use `adjoint. (a.coeffs)` and `adjoint (constant)` ? It is more readable IMO as it does not require to remember that ' is not transpose but adjoint

Reply:
That would create an `Adjoint{Vector}` rather than a `Vector`, so no, you can't do that.

Reply:
Actually, yes you can do it with the dot operator, didn't notice your dot.  Ok, I'll do that.